packages/components/pie-checkbox/src/checkbox.scss

```diff
@@ -1 +1,276 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+@keyframes checkboxCheck {
+    0% {
+        width: 0;
+        height: 0;
+        border-color: #fff;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    33% {
+        width: 8px;
+        height: 0;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    100% {
+        width: 8px;
+        height: 16px;
+        border-color: #fff;
+        border-bottom-right-radius: 2px;
+
+        transform: translate3d(0, -16px, 0) rotate(45deg);
+    }
+}
+
+@keyframes scaleDown {
+    0% {
+        transform: scale(0.7);
+    }
+
+    100% {
+        transform: scale(0);
+    }
+}
+
+@keyframes scaleUp {
+    0% {
+        transform: scale(0);
+    }
+
+    33% {
+        transform: scale(0.5);
+    }
+
+    100% {
+        transform: scale(1);
+    }
+}
+
+.c-checkbox {
+    --checkbox-height: 24px;
+    --checkbox-width: 24px;
+    --checkbox-radius: var(--dt-radius-rounded-a);
+    --checkbox-margin: 1px;
+    --checkbox-gap: var(--dt-spacing-b);
+    --checkbox-fz: var(--dt-font-body-l-size);
+    --checkbox-fw: var(--dt-font-weight-regular);
+    --checkbox-bg: var(--dt-color-container-default);
+    --checkbox-border-color: var(--dt-color-interactive-form);
+    --checkbox-content-color: var(--dt-color-content-default);
+    --checkbox-motion-easing: var(--dt-motion-easing-persistent-functional);
+    --checkbox-motion-timing-100: var(--dt-motion-timing-100);
+    --checkbox-motion-timing-200: var(--dt-motion-timing-200);
+
+    display: flex;
+    flex-direction: column;
+
+    input {
+        display: block;
+        position: absolute;
+        height: 1px;
+        width: 1px;
+        overflow: hidden;
+        clip: rect(1px, 1px, 1px, 1px);
+        clip-path: inset(1px);
+        white-space: nowrap;
+    }
+
+    label {
+        display: flex;
+        max-width: 100%;
+        white-space: nowrap;
+        cursor: pointer;
+    }
+
+    .c-checkbox-text {
+        display: inline;
+        flex: 1 1 auto;
+        align-self: center;
+        min-width: 0;
+        margin-inline-start: var(--checkbox-gap);
+        white-space: normal;
+        color: var(--checkbox-content-color);
+        font-size: var(--checkbox-fz);
+        font-weight: var(--checkbox-fw);
+    }
+
+    .c-checkbox-tick {
+        content: "";
+        display: flex;
+
+        flex: 0 0 auto;
+
+        width: var(--checkbox-width);
+        height: var(--checkbox-height);
+
+        margin: var(--checkbox-margin);
+
+        border: 1px solid var(--checkbox-border-color);
+        border-radius: var(--checkbox-radius);
+        background-color: var(--checkbox-bg);
+
+        transition: background-color var(--checkbox-motion-timing-100) var(--checkbox-motion-easing);
+
+        @media (prefers-reduced-motion: no-preference) {
+            animation: scaleDown var(--checkbox-motion-timing-100) var(--checkbox-motion-easing);
```

Comment:
is it not possible to just turn off the animation?

Reply:
Here the animation styles will apply only if there is no reduce motion preference set. And if it is set there won't be any animation.

Reply:
There is definitely an animation running when I enable prefers reduced motion

Reply:
I updated the styles and attached the screen recording for the reduced motion scenario